### PR TITLE
docs(scheduling): re-arrange docs related to scheduling, lineage, CLI

### PR DESCRIPTION
**Pull request by anshbansal**
## Checklist
- [x] The PR conforms to DataHub's [Contributing Guideline](https://github.com/linkedin/datahub/blob/master/docs/CONTRIBUTING.md) (particularly [Commit Message Format](https://github.com/linkedin/datahub/blob/master/docs/CONTRIBUTING.md#commit-message-format))
- [ ] Links to related issues (if applicable)
- [ ] Tests for the changes have been added/updated (if applicable)
- [x] Docs related to the changes have been added/updated (if applicable)

### CODE REVIEW

File: metadata-ingestion/schedule_docs/airflow.md [hunk 1]
```diff
@@ -0,0 +1,11 @@
+# Using Airflow
+
+If you are using Apache Airflow for your scheduling then you might want to use it for scheduling your ingestion receipes. To do that you can follow these steps
+- Create a receipe file e.g. `recipe.yml`
+- Ensure [DataHub CLI](../../docs/cli.md) is installed in your airflow environment
+- Create a sample DAG file like [`generic_recipe_sample_dag.py`](../src/datahub_provider/example_dags/generic_recipe_sample_dag.py). This will read your DataHub ingestion recipe file and run it.
```
Comment (shirshanka) [inline metadata-ingestion/schedule_docs/airflow.md:6]: How does the DAG get access to the recipe file when executing in the Airflow executor?

File: metadata-ingestion/schedule_docs/airflow.md [hunk 2]
```diff
@@ -0,0 +1,11 @@
+# Using Airflow
+
+If you are using Apache Airflow for your scheduling then you might want to use it for scheduling your ingestion receipes. To do that you can follow these steps
+- Create a receipe file e.g. `recipe.yml`
```
Comment (shirshanka) [inline metadata-ingestion/schedule_docs/airflow.md:4]: small typo : `receipe`

File: metadata-ingestion/schedule_docs/airflow.md [hunk 3]
```diff
@@ -0,0 +1,11 @@
...
+- Ensure [DataHub CLI](../../docs/cli.md) is installed in your airflow environment
+- Create a sample DAG file like [`generic_recipe_sample_dag.py`](../src/datahub_provider/example_dags/generic_recipe_sample_dag.py). This will read your DataHub ingestion recipe file and run it.
+- Deploy the DAG file into airflow for scheduling.
```
Comment (shirshanka) [inline metadata-ingestion/schedule_docs/airflow.md:7]: Is `deploy` the right verb here? maybe add another sentence that: Typically this involves checking in the DAG file into your dags repo for Airflow?

File: metadata-ingestion/schedule_docs/cron.md [hunk 4]
```diff
@@ -0,0 +1,28 @@
+# Using Cron
+
+To follow this guide you need to use [DataHub CLI](../../docs/cli.md).
+
+Assume you have a receipe file `/home/ubuntu/example.yml` on your machine
```
Comment (shirshanka) [inline metadata-ingestion/schedule_docs/cron.md:5]: name the file to be more descriptive? e.g.`/home/ubuntu/datahub_ingest/mysql_to_datahub.yml`

**Review by shirshanka — CHANGES_REQUESTED** [review]
Some small changes requested. 
need to change `receipe` to `recipe` throughout.

File: metadata-ingestion/schedule_docs/airflow.md (hunk 1 above)
Comment (anshbansal) [reply metadata-ingestion/schedule_docs/airflow.md:9]: added some details

**Review by shirshanka — APPROVED** [review]
LGTM

**Pull request merged**